addon/templates/components/d3-graph.hbs

```diff
@@ -1,6 +1,17 @@
-{{yield selection}}
 {{#if selection}}
-  {{#if graph}}
-    {{shhh (compute graph selection)}}
+  {{#if hasBlock}}
+    {{#if graph}}
```

Comment:
is there a nicer way to do this than using a triply-nested `if` statement? maybe we can use `ember-truth-helpers` here?